README.md

```diff
@@ -89,6 +89,22 @@ purposes), a self-signed certificate will be generated. Since the
 application runs in production mode, some kind of SSL cert is necessary
 in order for the application to function properly.
 
+### Using Custom Domain
+
+If you are using a custom domain, make sure your A record and nameservers
+are pointing to your newly created droplet. After verifying, you can now
+log into the image. When presented with the command-line interface for setup,
+exit the setup. There are a couple of nginx configs that need to be changed
+for setup to complete successfully. Edit `/etc/nginx/sites-available/default`
+by removing the SSL listen values, the ssl_certifcate and ssl_certificate_key
+values. Replace `_` with your domains in the server_name config.
+
+Double check that `WORKAREA_HOST` and `WORKAREA_CDN_HOST` are not present in
```

Comment:
Interesting, _not_ present? Should we remove these from the droplet?

Reply:
if these are present, the setup will not start up upon re-sshing into the image

Reply:
@thejoeyg did `certbot` work when you did this? the reason why these env vars are set are because Workarea relies on them to tell the application what `Workarea.config.host` and `Rails.configuration.asset_host` should be. otherwise I feel like you'd just get redirected to the domain it has set up in the configuration. One goal is we want this image to be workable without having to go in and edit configuration. Thinking I might need to make a change to the image that separates the nginx configs a bit better.

Reply:
@thejoeyg did `certbot` work when you did this? the reason why these env vars are set are because Workarea relies on them to tell the application what `Workarea.config.host` and `Rails.configuration.asset_host` should be. otherwise I feel like you'd just get redirected to the doain it has set up in the configuration. One goal is we want this image to be workable without having to go in and edit configuration. Thinking I might need to make a change to the image that separates the nginx configs a bit better.

Reply:
Yeah. This was the only way I got certbot to work. Once I restarted the setup, everything went as normal and upon finish those values were repopulated.